tests/modules/entity_module_bundle_plugin_test/src/Plugin/BundlePluginTest/BundlePluginTestInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\entity_module_bundle_plugin_test\Plugin\BundlePluginTest;
+
+use Drupal\entity\BundlePlugin\BundlePluginInterface;
+
+interface BundlePluginTestInterface extends BundlePluginInterface {
```

Comment:
nit: missing docblock. Do we need this or could the manager just look for the parent interface?